src/theme.ts

```diff
@@ -43,14 +59,33 @@ const graphColors = {
 // https://material-ui.com/customization/palette/#adding-new-colors
 declare module '@mui/material/styles/createPalette' {
   interface Palette {
-    cruYellow: Palette['primary'];
-    cruGrayDark: Palette['primary'];
-    cruGrayMedium: Palette['primary'];
-    cruGrayLight: Palette['primary'];
+    lemon: Palette['primary'];
+    yellow: Palette['primary'];
+    orange: Palette['primary'];
+    vermilion: Palette['primary'];
+    rose: Palette['primary'];
+    pink: Palette['primary'];
+    cerise: Palette['primary'];
+    ruby: Palette['primary'];
+    sky: Palette['primary'];
+    cyan: Palette['primary'];
+    turquoise: Palette['primary'];
+    navy: Palette['primary'];
+    mint: Palette['primary'];
+    green: Palette['primary'];
+    moss: Palette['primary'];
+    oliveDrab: Palette['primary'];
```

Comment:
Should these be secondary instead of primary?

Reply:
I resolved this by using an equivalent but more descriptive TypeScript type